### ISSUE TITLE
Search field not properly read in VPC (navigate with arrow keys) mode

### ISSUE THREAD

**Issue by ThaMoreira** [FreedomScientific/VFO-standards-support#690]
### Summary

If you have a Search field and use JAWS In VPC (navigate with arrow keys) mode, UI type for ‘search’ field is not read to the screen reader user and will not inform the user the information properly.

Example:

Find or quickly throw together some HTML with a search field. 
Turn on JAWS and enter the search field.
JAWS will not inform you all the information regarding the field.

### Expected result
JAWS will inform the user the search field by reading out something to the effect of "Search edit... Search field.. Type in text".

### Actual result
JAWS does not indicate all the information of the search field and reads out something to the effect of "search"
<img width="699" alt="search" src="https://user-images.githubusercontent.com/36004615/211061986-3060e1e3-3afc-4835-a48a-675cc4ac6f39.png">

### Example
[Search example] (https://jsbin.com/pofoxijetu/edit?html)

#### JAWS version and build number
JAWS version and build number
JAWS Version 2022.2204.20 ILM

**Comment by JAWS-test** [comment]
Here two different JAWS bugs would have to be described separately:
* input type=text and input type=search are both output as input field. The type=text and type=search are not distinguished. The missing distinction can be heard when navigating with TAB.
* with VPC the role of the input fields (no matter if type=text or type=search) is not output when automatically switching to form mode, but displayed on the Braille line.
